mod_semantic_map/SemMap map
string query_object
---
bool status
geometry_msgs/Point[] locations
float64[] pointProbabilities
int32[] bigObjectIDs
float64[] bigObjectProbability
